util/browser.js

```diff
@@ -34,17 +34,21 @@ export class BaseBrowser
 
     const args = this.chromeArgs(chromeOptions);
 
+    const geom = process.env.GEOMETRY.split("x");
+
+    const defaultViewport = {width: Number(geom[0]), height: Number(geom[1])};
```

Comment:
Perhaps naive question (I'm not super familiar with the `GEOMETRY` env var in this context, but will this always be set? Should we have a fallback?

Reply:
We always set it in the Dockerfile, but I suppose could also add a default here 'just in case', or not set it, if its missing for some reason..